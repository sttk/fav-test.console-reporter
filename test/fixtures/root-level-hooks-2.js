'use strict';

it('test3', function() {
  console.log('test3 run');
});

describe('suite4', function() {
  it('test4', function() {
    console.log('test4 run');
  });

  describe('suite5', function() {
    it('test5', function() {
      console.log('test5 run');
    });
  });
});
