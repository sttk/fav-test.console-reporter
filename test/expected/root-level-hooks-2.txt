

test3 run
  âœ“ test3

  suite4
test4 run
    âœ“ test4
    suite5
test5 run
      âœ“ test5


  3 passing (4ms)

